test/exitHandler.js

```diff
@@ -131,12 +131,37 @@ contract('ExitHandler', (accounts) => {
 
     describe('Start exit', async () => {
       it('Should allow to exit valid utxo', async () => {
-        const period = await submitNewPeriod([depositTx, transferTx]);
+        const period1 = await submitNewPeriod([depositTx, transferTx]);
```

Comment:
this change (the whole change in `exitHandler.js`) appears to be not relevant for the fix. I keep it here, because it is a good test 😆